src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -97,18 +99,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // get bundle directory
         $translationsPath = $rootPath.'/Resources/translations';
-        $output->writeln(sprintf('Generating "<info>%s</info>" translation files for "<info>%s</info>"', $input->getArgument('locale'), $currentName));
+        $output->title(sprintf('Generating "%s" translation files for "%s"', $input->getArgument('locale'), $currentName));
```

Comment:
This text looks more like a normal `->text()` thing instead of the command title.